### ISSUE TITLE
Add columns to authorized_user table for edi management

### ISSUE THREAD

**Issue by dlrueda** [sul-dlss/libsys-webforms#510]
edi_inv_manage
edi_inv_view

edi_inv_view can "View invoices"  
edi_inv_manage can "view invoices," do anything in the "updates" section, and "view errors"

**Comment by jgreben** [comment]
Fixed by #512

**Comment by jgreben** [comment]
This is completed for `@ltrxdev`, `@ltrxdev2` and `@ltrxprd`. 

App Admin permissions granted on the `@-dev` instances to: 
```
jgreben
dlrueda
sdoljack
rlunnon
linchang
```

App Admin permissions granted on the `@ltrxprd` instance to: 
```
jgreben
dlrueda
sdoljack
rlunnon
```
